bin/zowe-setup-certificates-prep.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+sed \
+  -e "s#HOSTNAME=#HOSTNAME=${ZOWE_EXPLORER_HOST}#" \
+  -e "s#IPADDRESS=#IPADDRESS=${ZOWE_IP_ADDRESS}#" \
+  -e "s#VERIFY_CERTIFICATES=#VERIFY_CERTIFICATES=${ZOWE_APIM_VERIFY_CERTIFICATES}#" \
+  -e "s#ZOSMF_KEYRING=#ZOSMF_KEYRING=${ZOWE_ZOSMF_KEYRING}#" \
+  -e "s#ZOSMF_USER=#ZOSMF_USER=${ZOWE_ZOSMF_USERID}#" \
+  -e "s#EXTERNAL_CERTIFICATE=#EXTERNAL_CERTIFICATE=${ZOWE_APIM_EXTERNAL_CERTIFICATE}#" \
+  -e "s#EXTERNAL_CERTIFICATE_ALIAS=#EXTERNAL_CERTIFICATE_ALIAS=${ZOWE_APIM_EXTERNAL_CERTIFICATE_ALIAS}#" \
+  -e "s#EXTERNAL_CERTIFICATE_AUTHORITIES=#EXTERNAL_CERTIFICATE_AUTHORITIES=${ZOWE_APIM_EXTERNAL_CERTIFICATE_AUTHORITIES}#" \
```

Comment:
Your sed logic isn't correct here - in order to overwrite the default values you need to include them in the sed script ie
`  -e "s#VERIFY_CERTIFICATES=#VERIFY_CERTIFICATES=${ZOWE_APIM_VERIFY_CERTIFICATES}#" \` should be `-e "  -e "s#VERIFY_CERTIFICATES=**true**#VERIFY_CERTIFICATES=${ZOWE_APIM_VERIFY_CERTIFICATES}#" \`